scripts/build/generateDocs.mjs

```diff
@@ -105,8 +105,17 @@ export function generateReadmeUrl(tag = 'master', variantOverride = '') {
   }
 
   const baseRepoUrl = 'https://raw.githubusercontent.com/AlaskaAirlines/WC-Generator'
-  if (tag !== 'master') {
-    return `${baseRepoUrl}/refs/tags/${tag}/componentDocs/README` + variantString + '.md';
+
+  // check if tag starts with 'vX' since our tags are `v4.0.0`
+  const isTag = branchOrTag.startsWith('v') &&
+    /^\d+\.\d+\.\d+(-.*)?$/.test(branchOrTag.slice(1));
```

Comment:
Consider adding a test for handling non main branch releases (e.g. `-beta.1`)

Reply:
this is covered in the new work!